packages/core/dao/WavesDAO.js

```diff
@@ -252,9 +200,11 @@ export default class WavesDAO extends EventEmitter {
 }
 
 //TODO WHY WE NEED SYMBOL AND ASSET DESCRIPTION IF SYMBOL IS ENOUGH
-function readBalanceValue (symbol, balance, asset = null) {
-  if (asset) {
-    return balance.assets[asset]
-  }
+function readBalanceValue (symbol, balance, /*asset = null*/) {
+  // @todo assets
```

Comment:
Useless code?

Reply:
No, it's a useful code. I created task for this todo